assets/pages/regular-contributions/helpers/ajax.js

```diff
@@ -201,6 +203,10 @@ function statusPoll(
 
   if (pollCount >= MAX_POLLS) {
     successfulConversion(participations);
+    ophan.sendInitialEvent(
```

Comment:
Instead of putting this events here, is it possible to add them in the root file of this page, i.e. the router>

Reply:
As offline discussed, we can't do this because if someone access directly the thank you page we will double-track. We should delete the auto-tracking feature form support ophan in a future PR.